truss/templates/control/control/helpers/inference_server_process_controller.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
+
+from helpers.context_managers import current_directory
+
+
+class InferenceServerProcessController:
```

Comment:
I'm a little confused what this class does and what `InferenceServerController` does. `InferenceServerController` is documented as "Controls the inference server process." but I'd imagine that this class, `InferenceServerProcessController`, would control the inference server process.

Reply:
Good point, let me clarify.